roles/reset/tasks/main.yml

```diff
@@ -264,7 +264,7 @@
   with_items:
     - "{{ kube_config_dir }}"
     - /var/lib/kubelet
-    - "{{ containerd_storage_dir }}"
+    - "{{ containerd_storage_dir | default('var/lib/containerd') }}"
```

Comment:
Why don't you specify `/var/lib/containerd` without `/` at the first word?

Reply:
Sorry I missed it